src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -243,8 +243,10 @@ function checkPrivilegesOfCustomForInstructor(instrNum) {
 }
 
 function showInstructorRoleModal(instrRole) {
-	checkPrivilegesOfRoleForModal(instrRole);
-	$('#tunePermissionsDivForInstructorAll').modal();
+	var isValidRole = checkPrivilegesOfRoleForModal(instrRole);
+	if (isValidRole) {
```

Comment:
Indentation is off, in fact I think this whole file seems to have garbled indentation, so it doesn't seem like it's your fault but since you're editing this portion might as well get it right before merging!